tests/polar_phasefield/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./action]
+    type = 'Exodiff'
+    input = 'action.i'
+    exodiff = 'action_out.e'
+  [../]
+  [./ic]
+    type = 'Exodiff'
+    input = 'ic.i'
+    exodiff = 'ic_out.e'
+  [../]
```

Comment:
May be handy to add the `no_action` example here to check its results against the `action`'s.

Reply:
added